models/UNetModule.py

```diff
@@ -2,6 +2,24 @@
 import tensorflow as tf
 from tensorflow import keras
 
+class Augment(tf.keras.layers.Layer):
+  def __init__(self, seed=42):
+    super().__init__()
+    # both use the same seed, so they'll make the same random changes.
+    self.augment_inputs = keras.Sequential([
+      tf.keras.layers.RandomFlip(mode="horizontal", seed=seed),
```

Comment:
Is there a benefit to doing horizontal and vertical sequentially vs using `mode="horizontal_and_vertical"`?

Reply:
No there isn't, I changed it to `mode="horizontal_and_vertical"`.